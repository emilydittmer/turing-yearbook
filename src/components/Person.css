.person-card{
  background-color: rgba(255, 255, 255, 0.95);
  border: 3px solid black;
  box-sizing: border-box;
  border-radius: 5%;
  max-width: 420px;
}

.card-photo{
  height: 200px;
  border-radius: 5%;
  max-width: 300px;
}

.name{
  
}