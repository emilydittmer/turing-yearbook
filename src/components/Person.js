import React from 'react';
import './Person.css';

const Person = ({id, name, quote, superlative, photo}) => {
  return (
    <article className="person-card">
      <h2 className='name'>{name}</h2>
      <img src= {photo} alt="Person" className="card-photo"/>
      <p className='quote'>"{quote}"</p>
      <p className='super'>{superlative}</p>
    </article>
  )
}

export default Person;
