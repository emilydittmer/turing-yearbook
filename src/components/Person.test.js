import React from 'react';
import { shallow, mount} from 'enzyme';
import Person from './Person';

describe('Person', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    let wrapper = shallow(
      <Person 
        name='Emily'
        quote='Great Quote'
        superlative='Most likely to be amazing'
      />
    );

    expect(wrapper).toMatchSnapshot()
  })
})