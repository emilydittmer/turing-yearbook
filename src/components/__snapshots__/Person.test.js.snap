// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person should match the snapshot with all data passed in correctly 1`] = `
<article
  className="person-card"
>
  <h2
    className="name"
  >
    Emily
  </h2>
  <img
    alt="Person"
    className="card-photo"
  />
  <p
    className="quote"
  >
    "
    Great Quote
    "
  </p>
  <p
    className="super"
  >
    Most likely to be amazing
  </p>
</article>
`;
